pkgs/by-name/su/supergfxctl-plasmoid/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, cmake
+, kdePackages
+}:
+
+stdenv.mkDerivation rec {
+  pname = "supergfxctl-plasmoid";
+  version = "2.0.0";
+
+  src = fetchFromGitLab {
+    owner = "jhyub";
+    repo = "supergfxctl-plasmoid";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-m3NmbFD9tqqCyiQgMVRNtlCZy7q+rMCsWgtds1QdOrE=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = [
+    kdePackages.wrapQtAppsHook
```

Comment:
Just noticed this, but this should go in `nativeBuildInputs`

Reply:
gotcha, I'll correct that as well
also, sorry, I'm trying to look into the rebasing and squashing to try and get it right, my bad

Reply:
I'm having some trouble with the whole commit squashing
if it's not too much trouble, if and when at all possible, could you help me out?